src/backend/distributed/test/metadata_sync.c

```diff
@@ -14,6 +14,7 @@
 #include "fmgr.h"
 
 #include "catalog/pg_type.h"
+#include "distributed/listutils.h"
 #include "distributed/metadata_sync.h"
 #include "distributed/test_helper_functions.h" /* IWYU pragma: keep */
```

Comment:
Removed all #includes of `test_helper_functions.h` and didn't get any complains. Weird.